Ujjwal.java

```diff
@@ -0,0 +1,39 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+class Ujjwal {
+    public static void main(String... k) {
+        Scanner scan = new Scanner(System.in);
+        String str;
+        int test;
+        str = scan.nextLine();
+        test = scan.nextInt();
+
+        ArrayList<Integer> arrayOfIndex = new ArrayList<>();
+        ArrayList<String> arrayOfTaskStrings = new ArrayList<>();
+
+        while (test > 0) {
+            arrayOfTaskStrings.add(scan.next());
+            test--;
+        }
+        for (String indvidualTask : arrayOfTaskStrings) {
+            int index = str.indexOf(indvidualTask, 0);
```

Comment:
Try not to use predefined methods like indexOf(). Try to write language independent code. indexOf() will lead to TLE on large test cases.